config/routes.rb

```diff
@@ -1,3 +1,13 @@
 Rails.application.routes.draw do
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  namespace :api do
+    namespace :v1 do
+      # Sensors
+      get '/users/:id/gardens/:id/sensors', to: 'sensors#index'
+      get '/users/:id/gardens/:id/sensors/:id', to: 'sensors#show'
+      post '/users/:id/gardens/:id/sensors', to: 'sensors#create'
+      patch '/users/:id/gardens/:id/sensors/:id', to: 'sensors#update'
+      delete '/users/:id/gardens/:id/sensors/:id', to: 'sensors#delete'
```

Comment:
@lkriffell Looks like these all could be namespaced.

Reply:
I agree with @leahriffell. They could be refactored using `resources`

Reply:
On it!